products/public-websites/cerner/postmortem/missing-content-schedule-appt-page.md

```diff
@@ -0,0 +1,103 @@
+
+An [off-schedule deploy was requested](https://dsva.slack.com/archives/CBU0KDSB1/p1597695472119100?thread_ts=1597695207.118200&cid=CBU0KDSB1) at 4:17pm EST, which was then executed at 5:52pm EST. Following the fix being deployed at that time, traffic resumed as normal to VAOS.
+
+## Background
+
+A request was made to update the current content on the page [www.va.gov/health-care/schedule-view-va-appointments](https://www.va.gov/health-care/schedule-view-va-appointments/) with [this GitHub comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12446#issuecomment-674324572).
+
+> we separately need to update the legacy content for this schedule and view appointments page for the Monday 8.17 deploy. The VA appointments tool is adding community care appointments that day, but we won't be quite ready to just switch to the unauthenticated Cerner content. I'm attaching the needed changes to the legacy page. Can you update these or show me how to update Monday morning? Thank you!
+
+[This request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12446#issuecomment-674324572)
+
+## Detection
+_How was the issue found?_ It was found by Danielle Thierry.
+
+![Discovery message](./discovery.png)
+
+_Date/time of discovery_ 4:05pm EST
+
+## Impact
+
+_What was the impact on Veterans, business and team?_
+
+All traffic to VAOS from [www.va.gov/health-care/schedule-view-va-appointments](https://www.va.gov/health-care/schedule-view-va-appointments/) ceased from 3:15pm EST to 5:52pm EST.
+
+## Root Causes
+
+_What caused this situation?_
+
+This situation was caused by 2 3-digit TTY 711 numbers that were using the `formation-react` `<Telephone />` component. These components required a prop called `pattern` that was not present, causing a runtime error, which resulted in the React widget not rendering.
+
+## Resolution
+_What steps were taken to resolve the issue?_ A PR that added the `pattern` prop to both `<Telephone />` components and an off-schedule deploy.
+
+_Date/time of resolution_ At 5:52pm EST on August 17, 2020.
+
+## Documentation links
+
+- [Original PR with bug](https://github.com/department-of-veterans-affairs/vets-website/pull/13882)
+- [PR with bug fix](https://github.com/department-of-veterans-affairs/vets-website/pull/13896)
+- [Slack discussion in #vfs-platform-support with request of off-schedule deploy](https://dsva.slack.com/archives/CBU0KDSB1/p1597695207118200)
+- [Slack discussion in #vsa-public-websites](https://dsva.slack.com/archives/C52CL1PKQ/p1597695037082300)
+
+## Action Items
+
+| Description                    | Type    | Owner        | Issue # |
+| ------------------------------ | ------- | ------------ | ------- |
+| Brainstorm alternatives of storing authenticated content in React widgets as opposed to Drupal | Frontend | Public Websites / CMS | Ticket not created yet |
```

Comment:
I still don't understand who really owns or has say in how this content is written because I think there are issues with having a whole page of content transform based on user status. I've long wondered if this is acceptable for screen readers. 

Anyway, point being that I think it's okay to store authenticated content in React as long as that content is "containerized" so that only a certain subset of the page transforms.